### DISCUSSION TITLE
Avoid deployment of multiple Framework-DLLs

### DISCUSSION THREAD

**Discussion by rkukasch** [stephanstapel/ZUGFeRD-csharp#145] [Ideas]
If you use the nuget-package in an .NET 4.6.1 project, multiple .NET assemblies have to be deployed.
To avoid this I suggest to change ZUGFeRD.csproj:

add 
`<TargetFrameworks>netstandard2.0;net461</TargetFrameworks>`
and remove 
`<TargetFramework>netstandard2.0</TargetFramework>`

**Comment by stephanstapel** [comment]
Hi,

thanks for the idea. Honestly I don't have too many experience with nuget and the more sophisticated usage of it.
In case you are more into this, two questions:

* The assembly is compiled as a netstandard2.0 assembly. So I assume that it really needs those additional dependencies since it is not net461. Am I right?
* I'd probably better add all of net461...net48, right?

**Reply by rkukasch** [reply]
- yes, it needs these assemblies
- adding only net461 is ok, because 4.6.1 assemblies are forward compatible with framework 4.8
On my machine, building the project with the suggested change generates to subfolders in bin: net461 and netstandard2.0.
When I generate the nuget package s2industries.ZUGFeRD.6.0.0.nupkg, it contains both versions of the dll.

Sorry for the late answer and thank you for the excellent work :)

**Reply by stephan-nowecor** [reply]
Thanks, I have added this to the project file and made it ready to pack the nuget package from there.

If you use the project commercially, I'd be more than happy for github sponsoring :)
